<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WheatherChannel</class>
 <widget class="QMainWindow" name="WheatherChannel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WheatherChannel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Temperature</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="getTemperature">
             <property name="text">
              <string>Mesurer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="autoTemp">
             <property name="title">
              <string>Mesures automatiques</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Temps entre deux mesures (en s) :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="freqAutoTemperature">
                <property name="maximum">
                 <number>100000000</number>
                </property>
                <property name="value">
                 <number>5</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="updateFreqAutoTemperature">
                <property name="text">
                 <string>Mettre Ã  jour</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="outputText"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QGraphicsView" name="visualizer"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
